omp/solver/lower_trs_kernels.cpp

```diff
@@ -58,10 +58,17 @@ namespace omp {
 namespace lower_trs {
 
 
+void clear(std::shared_ptr<const OmpExecutor> exec)
+{
+    // This clear kernel is here to allow for a more sophisticated
```

Comment:
It is missing what the "more sophisticated implementation" can actually do with this function.
```suggestion
    // This clear kernel is here to allow cleanup for a more sophisticated
```